node/src/service.rs

```diff
@@ -518,7 +522,7 @@ pub fn new_full<
 							log::warn!("API error getting threshold: {:?}", e);
 							U512::zero()
 						});
-					let nonces_to_mine = 3000u64;
+					let nonces_to_mine = 3_000_000u64;
```

Comment:
why mine 3M nonces? might take too long. Also I changed this to 300 recently, not sure why it is back to 3000.

Reply:
Ok lets change it back to 300, although, I have another PR now which also does that.

Reply:
Ok nevermind... please merge main into this, set it back to 300 and added the threading fix in main

Reply:
Actually I merged it, was only 1 line